components/auth/otp-form.tsx

```
@@ -55,6 +70,78 @@ export default function OTPForm() {
+                    // Cast to OTP credential interface
+                    (
+                        navigator.credentials as {
+                            get(options: OTPCredentialRequestOptions): Promise<OTPCredential | null>;
+                        }
+                    )
+                        .get({
+                            otp: { transport: ["sms"] },
+                            signal
+                        })
+                        .then((credential: OTPCredential | null) => {
+                            if (credential && credential.code) {
```

Biome (1.9.4)

[error] Change to an optional chain.

Unsafe fix: Change to an optional chain.


(lint/complexity/useOptionalChain)